initialization/metadata.go

```diff
@@ -11,7 +11,18 @@ import (
 
 const MetadataFileName = "postdata_metadata.json"
 
+func getEffectiveDir(dir string) string {
+	envDir := os.Getenv("POST_METADATA_DIR")
+	fmt.Println("Got postmeta dir of env:", envDir)
+	if envDir != "" {
+		return envDir
+	}
+	return dir
```

Comment:
How about using `os.LookupEnv` instead:
```suggestion
	if value, ok := os.LookupEnv("POST_METADATA_DIR"); ok {
        	return value
	}
	return dir
```